tests/no_inst_fleet_unit_tests.py

```diff
@@ -0,0 +1,311 @@
+            },
+            {
+                "basis": "mass",
+                "name": "spent_uox",
+                "nuclide": [{"comp": "50", "id": "Kr85"}, {"comp": "50", "id": "Cs137"}]
+            }
+        ],
+        "facility": [{
+            "config": {"Source": {"outcommod": "fuel",
+                                  "outrecipe": "fresh_uox",
+                                  "throughput": "100",
+                                  "source_record_supply": "fuel"}},
+            "name": "source"
+        },
+            {
+            "config": {"Sink": {"in_commods": {"val": "spent_uox"},
+                                "max_inv_size": "1e9",
+                                "sink_record_demand": "fuel_cap"}},
+            "name": "sink"
+        },
+            {
+            "config": {
+                "Reactor": {
+                    "assem_size": "100",
+                    "cycle_time": "10",
+                    "fuel_incommods": {"val": "fuel"},
+                    "fuel_inrecipes": {"val": "fresh_uox"},
+                    "fuel_outcommods": {"val": "spent_uox"},
+                    "fuel_outrecipes": {"val": "spent_uox"},
+                    "n_assem_batch": "1",
+                    "n_assem_core": "1",
+                    "power_cap": "1",
+                    "refuel_time": "0",
+                    "reactor_fuel_demand": "fuel_reactor"
+                }
+            },
+            "name": "reactor"
+        }]
+    }
+}
+
+
+""" Test naming convention: 
+Test-[alphabet]-[changetype]-[numbering]
+alphabet
+  - a: only source facility 
+  - b: source and reactor facilities 
+  - c: source, reactor and sink facilties 
+change type 
+  - const: constant demand of commodity that drives deployment
+  - growth: growing demand of commodity that drives deployment
+  - decl: declining demand of commodity that drives deployment
+numbering 
+  - if test falls under same alphabet and change type category, they
+    are numbered 
+"""
+
+
+""" TOLERANCES """
+# No. of timesteps accepted for supply to catch up with initial demand
+catchup_tolerance = 12 
+
+# No. of facility throughputs for acceptable diff btwn of supply and demand
+facility_tolerance = 1
+
+# demand curve function 
+
+def demand_curve(initial_demand,growth_rate,x_point):
+    y_point = initial_demand*(1+growth_rate)**(x_point/12)
+    return y_point 
+
+""" Test a-const-1 
+        - a: only source facility 
+        - const: constant demand of fuel commodity that drives deployment 
+        - 1: first test of a-const type 
+"""
+# Configuring it for this test instance 
+test_a_const_1_temp = copy.deepcopy(TEMPLATE)
+test_a_const_1_temp["simulation"].update({"region": {
+    "config": {"NullRegion": "\n      "},
+    "institution": {
+        "config": {
+            "NOInst": {
+                "calc_method": "arma", 
+                "commodities": {"val": ["fuel"]}, 
+                "demand_std_dev": "0.0", 
```

Comment:
It might be interesting to run this test with a positive value for the demand_std_dev. Should make it so it only under predicts at the beginning of the simulation.